about.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/about.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Readex+Pro:wght@200&display=swap" rel="stylesheet">
+  <title>About</title> 
+  
+</head>
+<body>
+  <header>
+    <div>
+      <div class="grid-container"> 
+        <div class="grid-spacing"> <a href="index.html">Home</a></div>
+        <div class="grid-spacing"> <a href="about.html">About</a></div>
+        <div class="grid-spacing"> <a href="portfolio.html">Portfolio</a></div>
+      </div>
+    </div>
+  </main>
```

Comment:
This looks like a stray closing tag (running through a validator can catch things like this).